lua/lv-utils/init.lua

```diff
@@ -1,5 +1,61 @@
 local lv_utils = {}
 
+-- recursive Print (structure, limit, indent)
+local function rPrint(s, l, i)
+  l = l or 100 -- default item limit
+  i = i or "." -- indent string
+  if l < 1 then
```

Comment:
Can you explain what the limit is for ?

Reply:
just to make sure if my code has issues it won't loop forever, but might remove it after refactoring